cms-xcache/sbin/renew-proxy

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+/usr/local/sbin/fix_certs.sh
+su xrootd /usr/libexec/xcache/renew-proxy --voms cms
```

Comment:
Let's rename this to `cms-xcache/image-config.d/20-cms-generate-proxy.sh` and drop it in the proper dir in the Dockerfile